api/wip-files/examples/2024-06-10-preview/HypershiftClusters_CreateOrUpdate.json

```diff
@@ -0,0 +1,76 @@
+{
+  "title": "HypershiftClusters_CreateOrUpdate",
+  "operationId": "HypershiftClusters_CreateOrUpdate",
+  "parameters": {
+    "api-version": "2021-10-01-preview",
+    "subscriptionId": "11809CA1-E126-4017-945E-AA795CD5C5A9",
+    "resourceGroupName": "rgopenapi",
+    "employeeName": "9KF-f-8b",
+    "resource": {
+      "properties": {
+        "age": 30,
+        "city": "gydhnntudughbmxlkyzrskcdkotrxn",
+        "profile": "ms"
```

Comment:
these don't make any sense, do we need to have these example files at all? or is there a way to have them be more accurate?

Reply:
I have removed the examples altogether, it was too confusing. I hoped to reuse these autogenerated ones, but they diverged too much.